packages/typespec-ts/test/unit/modelsGenerator.spec.ts

```diff
@@ -2894,6 +2894,94 @@ describe("Input/output model type", () => {
         `
       });
     });
+
+    it("should generate template model successfully even without @friendlyName", async () => {
```

Comment:
@jiaodi could you add this case into modular also under `modularUnit/scenarios/models/template` folder?

Reply:
resolved